book/website/communication/citable/citable-linking.md

```diff
@@ -4,57 +4,62 @@
+But if you are hesitant about opening up your dataset before the article is published, you can either publish the dataset under restricted access, under embargo, or 
+reserve a DOI in advance and publish the Research Object later.
 
 ## 2. Keep the Research Objects closed
 
 ### Publish the Research Objects under restricted access
 
 Publishing your data under restricted access means that the data is uploaded to the repository, but you're only providing access to individuals that request access or you can provide access through sharing a private link. 
 * You decide with whom to share the private link and whether individuals that request access will obtain access.
-    * You can share the data with reviewers
+    * You can share the data with reviewers.
 * Your metadata (information about the data) is publicly available.
-* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`) 
+* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`). 
 * After your article is published: 
-    * Add the DOI of the publication to the dataset 
+    * Add the DOI of the publication to the dataset; 
     * Change the access permission rights to public (see for example [Zenodo](https://zenodo.org/)).
 
 ### Publish the Research Objects under embargo
 
 An embargo means that you have already uploaded the data to the repository, but are not providing access to the data until a certain date. 
 * No one has access to your data until the embargo period has expired (you can define this period), or until you changed the access permissions yourself. 
-    * You cannot share the data with reviewers
+    * You cannot share the data with reviewers.
 * Your metadata (information about the data) is publicly available.
-* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`) 
+* You can provide the correct DOI in the paper's **Data Availability Statement** and in the reference list (see {ref}`Citing Data<cm-citable-cite-data>`). 
 * After your article is published: 
-    * Add the DOI of the publication to the dataset 
+    * Add the DOI of the publication to the dataset; 
     * Change the access permission rights if the embargo has not yet expired (see for example [Zenodo](https://zenodo.org/)).
 
 ### Reserve a DOI and publish later
 
-Several data repositories allow you to reserve a DOI (see for example [Zenodo](https://zenodo.org/), [Figshare](https://figshare.com/))
+Several data repositories allow you to reserve a DOI, such as [Zenodo](https://zenodo.org/) and [Figshare](https://figshare.com/).
```

Comment:
From a readability point of view, having the examples right after the term "data repositories" works a bit better. But no strong feelings, really.

Reply:
Done!